lib/install/config/loaders/installers/elm.js

```diff
@@ -4,11 +4,11 @@ const { env } = require('../configuration.js')
 const elmSource = path.resolve(process.cwd())
 
 const loaderOptions = () => {
-    if (env.NODE_ENV === "production") {
-        return `elm-webpack-loader?cwd=${elmSource}`
-    }
+  if (env.NODE_ENV === 'production') {
```

Comment:
@gauravtiwari, @skarger Ah, yes, sorry about that. Read that incorrectly. Makes sense.